wPortal/FolderPermissions.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Folder Permissions
+filename: "FolderPermissions.md"
+layout: custom
+keywords: []
+headings: ["Overview", ""]
+links: ["/wPortal/INTERJECT-Roles.html"]
+image_dir: "FolderPermissions"
+images: [
+    {file: "FolderPermissionsPage", type: "png", site: "Portal", cat: "Folder permissions", sub: "", report: "", ribbon: "", config: ""},
+    {file: "FolderCustomRole", type: "png", site: "Portal", cat: "Folder permissions", sub: "", report: "", ribbon: "", config: ""}
+]
+description: 
+---
+* * *
+
+## Overview
+
+The Folder Permissions page allows you to assign roles to folders in the Report Library. The role count shows how many roles have been assigned to that particular folder. Folders with roles assigned to it are only viewable by users with those roles. If no roles have been assigned to the folder, then the they are viewable to all roles.
```

Comment:
second sentence should be "folders with roles assigned to "them"... or else it should be "A folder with..."